woodwork/tests/utils/test_accessor_utils.py

```diff
@@ -45,6 +46,19 @@ def test_init_series_with_invalid_type(sample_df):
             init_series(input_)
 
 
+def test_init_series_with_np_array(sample_series_pandas):
+    inputs = [np.array([['a', 'b'], ['b', 'c']]), np.array(['a', 'b', 'c', 'a'])]
+    for input_ in inputs:
+        if input_.ndim == 1:
+            series = init_series(input_)
+            series2 = init_series(sample_series_pandas)  # Sample series panda contains ['a','b','c','a']
+            assert series.equals(series2)
```

Comment:
We should also test that the Woodwork typing information is as expected.

Reply:
Also, as this test grows, it'll probably make sense to separate the multi dimensional array case into its own test. There's not really any shared code between the two tests, so it's better to separate them out so we can potentially see two different bugs at the same time (as opposed to a bug from the first element masking one from the second)